TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Knative Trademark committee"
+linkTitle: "Trademark committee"
+weight: 50
+type: "docs"
+aliases:
+  - /contributing/trademark-committee/
+---
+
+The Knative Trademark Committee (KTC)
+[advises the trademark owner on the usage and enforcement of the Knative mark](./GOVERNANCE.md).
+It is a separate body from the [Steering Committee](./STEERING-COMMITTEE.md) to
+better represent the needs of vendors and other companies which market products
+in conjuction with the Knative mark.
+
+The governance of Knative is an open, living document, and will continue to
+evolve as the community and project change. We expect over time we will adapt
+the way we run this committee, based on feedback from the community.
+
+- [Charter](#charter)
+- [Delegated Authority](#delegated-authority)
+- [Committee Meetings](#committee-meetings)
+- [Committee Members](#committee-members)
+- [Decision Process](#decision-process)
+- [Getting in touch](#getting-in-touch)
+- [Composition](#composition)
+  - [Maintaining the Trademark Committee](#maintaining-the-trademark-committee)
+
+## Charter
+
+1. Advise the Trademark owner on the definition and enforcement of the "Knative"
+   trademark usage and branding guidelines.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
```

Comment:
I think this is easier to understand as a power if we just say 'Review and approve conformance rules and tests.' If we think more clarification of the use of these powers is needed, we can add a section - but ideally the charter would speak only to the powers of each body without implementation details/elaboration.

Reply:
I added this due to the discussion that @ronavn and @julz had around a hypothetical "Knative Extension Functions" framework which wanted to publish a conformance suite for a component which was not in the "exported set of trademarks" managed by the KTC.

(Thinking of the brands the KTC manages as module exports is an analogy that just occurred to me, but it feels accurate.)

Reply:
I've no objection to pulling this out into a section that explains the goals. Does that work?